### DISCUSSION TITLE
Feature request: share lists

### DISCUSSION THREAD

**Discussion by mit-olyk** [mastodon/mastodon#20986] [General]
The ability to share features is a great way to allow people who have migrated en masse from other sites not to be named to easily find each other and connect.

Considering the decentralized nature of Mastodon, it would also require lists to migrate with users should they migrate to another server. 

Also, for these shared lists, I think it would be important for members of the lists to opt-out should they not want to participate in a particular shared list.

One person would be the owner/Admin of the list, but they would have the authority to identify others within the list to have administrative authority as well.

**Comment by mit-olyk** [comment]
I didn't do enough searching. I looked in discussion thread, but not issues, where this suggestion is being tracked. https://github.com/mastodon/mastodon/issues/19882

**Comment by Terra100f** [comment]
Would love to have this feature -- to keep your list and to share lists. I used lists extensively in Twitter -- turning on some to be public and others private. I know lots of people who followed my public lists. And both my public and private lists made it efficient for me with my time to review a group of people's posts.
